tests/unit/data/base/test_video.py

```diff
@@ -0,0 +1,50 @@
+"""Unit Tests - Base Video Datamodules."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from anomalib.data import AnomalibDataModule
+import random
+from .test_datamodule import _TestAnomalibDataModule
+
+
+class _TestAnomalibVideoDatamodule(_TestAnomalibDataModule):
+    def test_get_item_returns_correct_keys_and_shapes(self, datamodule: AnomalibDataModule) -> None:
+        """Test that the datamodule __getitem__ returns image, mask, label and boxes."""
+
+        # Randomly select a subset of the dataset.
+        subsets = ["train", "val", "test"]
```

Comment:
Do we need random subsets, how much extra time would it take if we tests all the subsets.